### DISCUSSION TITLE
Presentation at OpenGeoHub

### DISCUSSION THREAD

**Discussion by Robinlovelace** [ropensci/osmextract#283] [General]
Heads-up @agila5 and anyone else interested, I've drafted a talk on using large OSM datasets for OpenGeoHub 2023, comments/suggestions welcome: https://github.com/robinlovelace/opengeohub2023#processing-large-openstreetmap-datasets-for-geocomputational-research

**Comment by agila5** [comment]
Hi Robin! I'm glad to see interest regarding `osmextract`. Also happy to check the updated draft/slides in the next weeks. Text me if you need help with some parts.
